CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.1.1
+- Fix file upload chunk.
```

Comment:
@WMKhant we need a better description of the change here. I suggest "Fix file upload by chunks method to use the right sequence of parameters"

Reply:
Also, please update the name of the PR accordingly.

Reply:
@myfoxtail update description

Reply:
Thank you @WMKhant .